straxen/plugins/afterpulse_processing.py

```diff
@@ -0,0 +1,345 @@
+            (('Height of hit in ADC counts', 'height'),
+              '<i4'),
+            (('Height of hit in PE', 'height_pe'),
+              '<f4'),
+            (('Delay of hit w.r.t. LED hit in same WF, in samples', 'tdelay'),
+              '<i2'),
+            (('Internal (temporary) index of fragment in which hit was found', 'record_i'),
+              '<i4'),
+            (('Index of sample in record where integration starts',
+              'left_integration'),
+              np.int16),
+            (('Index of first sample beyond integration region',
+              'right_integration'),
+              np.int16),
+            (('ADC threshold applied in order to find hits',
+              'threshold'),
+              np.float32),
+           ]
+
+
+@export
+@strax.takes_config(
+                    strax.Option('gain_model',
+                                 help='PMT gain model. Specify as (model_type, model_config)',
+                                ),
+                    strax.Option('n_tpc_pmts',
+                                 type=int,
+                                 help="Number of PMTs in TPC",
+                                ),
+                    strax.Option('LED_window_left',
+                                 default=50,
+                                 help='Left boundary of sample range for LED pulse integration',
+                                ),
+                    strax.Option('LED_window_right',
+                                 default=100,
+                                 help='Right boundary of sample range for LED pulse integration',
+                                ),
+                    strax.Option('baseline_samples',
+                                 default=40,
+                                 help='Number of samples to use at start of WF to determine the baseline',
+                                ),
+                    strax.Option('hit_min_amplitude',
+                                 track=True,
+                                 default=('hit_thresholds_tpc', 'ONLINE', True),
+                                 help='Minimum hit amplitude in ADC counts above baseline. '
+                                      'Specify as a tuple of length n_tpc_pmts, or a number,'
+                                      'or a string like "pmt_commissioning_initial" which means calling'
+                                      'hitfinder_thresholds.py'
+                                      'or a tuple like (correction=str, version=str, nT=boolean),'
+                                      'which means we are using cmt.',
+                                ),
+                    strax.Option('hit_min_height_over_noise',
+                                 default=4,
+                                 help='Minimum hit amplitude in numbers of baseline_rms above baseline.'
+                                      'Actual threshold used is max(hit_min_amplitude, hit_min_height_over_noise * baseline_rms).',
```

Comment:
So for regular data this is not really used actually. And I would think as you are using triggered data this should be needed even less. Or are you suspecting that the LED pulser makes some noise?